examples/user_guide/03-Customizing_Plots.ipynb

```diff
@@ -63,13 +63,87 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%output size=150\n",
-    "%%opts Curve  [height=100 width=600 xaxis=None tools=['hover']]\n",
+    "%%opts Curve [height=100 width=600 xaxis=None tools=['hover']]\n",
     "%%opts Curve (color='red' line_width=1.5)\n",
     "%%opts Spikes [height=100 width=600 yaxis=None] (color='grey' line_width=0.25)\n",
-    "curve  = hv.Curve( spike_train, 'milliseconds', vdims='Hertz')\n",
+    "curve = hv.Curve( spike_train, 'milliseconds', vdims='Hertz')\n",
+    "spikes = hv.Spikes(spike_train, 'milliseconds', vdims=[])\n",
+    "(curve + spikes).cols(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here it is, again, with the `.options` method, which is the simplest way of customizing because it will automatically deduce whether an option is a style, plot, or norm option. In addition, `.options` is portable across notebook environments and `.py` scripts, and also will be saved on export, unlike `%%opts`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "curve_opts = dict(height=100, width=600, xaxis=None, tools=[\n",
+    "                  'hover'], color='red', line_width=1.5)\n",
+    "spike_opts = dict(height=100, width=600, yaxis=None,\n",
+    "                  color='grey', line_width=0.25)\n",
+    "\n",
+    "curve = hv.Curve(spike_train, 'milliseconds', 'Hertz')\n",
+    "spikes = hv.Spikes(spike_train, 'milliseconds', [])\n",
+    "\n",
+    "(curve.options(**curve_opts) + spikes.options(**spike_opts)).cols(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When using ``.options`` to apply options directly to an individual object we do not have to explicitly declare which object the options apply to, however often it is useful to set options on a composite object. In these cases the options can be declared as a dictionary of the type name and the options. The code below is therefore equivalent to the syntax we used above:"
```

Comment:
@philippjfr, does .options only accept a type name, or does it also accept a fully qualified type.group.label string?

Reply:
``type.group.label`` is valid.

Reply:
Ok, that's what I thought.  The text needs to be updated here to reflect that.